### PR TITLE
Custom meta frontend

### PR DESCRIPTION
**Pull request by francawinter**

### CODE REVIEW

File: frontend/src/Page/index.js [hunk 1]
```diff
@@ -133,6 +133,21 @@ function Page(props) {
...
   }
 
+  let custom = [];
+
+  for (let title in props.custom) {
+    // Display only one list item
+    let listContent = props.custom[title];
+
+    // Display more than one list item
+    if (typeof(listContent) == 'object') {
```
Comment (mariuswilms) [inline frontend/src/Page/index.js]: Please use strict comparison `===` as the default.
```suggestion
    if (typeof(listContent) === 'object') {
```

File: frontend/src/Page/index.js [hunk 2]
```diff
@@ -133,6 +133,21 @@ function Page(props) {
     authors = <Meta title={title}>{authorLinks}</Meta>;
   }
 
+  let custom = [];
+
+  for (let title in props.custom) {
```
Comment (mariuswilms) [inline frontend/src/Page/index.js]: The code block basically transforms each item in `props.custom` in a corresponding new item. Using `Array.map` will better express this procedure.

Comment (ChristophLabacher) [reply frontend/src/Page/index.js]: `props.custom` is an object, so `Array.map` is not an option, if I’m not mistaken …

File: frontend/src/Page/index.js (hunk 1 above)
Comment (ChristophLabacher) [reply frontend/src/Page/index.js]: Not sure if checking if this is an object is the right thing to do here – shouldn’t we use `Array.isArray(obj)` in order to check if it is an array?

File: frontend/src/Page/index.js (hunk 2 above)
Comment (mariuswilms) [reply frontend/src/Page/index.js]: `Object.entries()` is your friend https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

File: frontend/src/Page/index.js (hunk 1 above)
Comment (mariuswilms) [reply frontend/src/Page/index.js]: Thanks for catching yes, this is indeed the better approach.

**Comment by francawinter** [comment]
Hi @mariuswilms,
thank you for your feedback. I implemented it and hopefully it's more accurate now!

**Comment by francawinter** [comment]
Hello @mariuswilms, 
I improved the code styling. Thanks again for sharing your knowledge!

**Comment by mariuswilms** [comment]
I've squashed your commits and merged them as 830af665bed7d32336e73646665b401a07147205, thanks!

**Pull request closed without merging**